packages/route-optimization-jsprit/gradlew

```diff
@@ -82,6 +82,7 @@ esac
 
 CLASSPATH=$APP_HOME/gradle/wrapper/gradle-wrapper.jar
 
+
```

Comment:
```suggestion
```
unless needed

Reply:
Most likely not, it's just autogenerated by gradle itself

Reply:
ah interesting :+1:

Reply:
Thanks for helping out here! I was also running the tests in a container with OpenJDK17 and all tests got green.

Reply:
Sure, but you didn't want to merge it? You closed the PR :)

Reply:
I think we need Gradle 7.0 for OpenJDK17, IIRC 16 is only supported by 6.9 and/or 7.0